---

Fuck the annoying flash when you do a screenshot in GNOME

---
diff --git a/gnome-shell/js/ui/screenshot.js b/gnome-shell/js/ui/screenshot.js

--- a/gnome-shell/js/ui/screenshot.js	2023-10-26 09:37:37.247661812 +0800
+++ b/gnome-shell/js/ui/screenshot.js	2023-10-26 09:39:54.212736643 +0800
@@ -2937,16 +2937,8 @@
     }
 
     fire(doneCallback) {
-        this.set({ visible: true, opacity: 255 });
-        this.ease({
-            opacity: 0,
-            duration: FLASHSPOT_ANIMATION_OUT_TIME,
-            mode: Clutter.AnimationMode.EASE_OUT_QUAD,
-            onComplete: () => {
-                if (doneCallback)
-                    doneCallback();
-                this.destroy();
-            },
-        });
+        if (doneCallback)
+            doneCallback();
+        this.destroy();
     }
 });
